esmvalcore/cmor/check.py

```diff
@@ -696,6 +695,11 @@ def _check_requested_values(self, coord, coord_info, var_name):
                 cmor_points = [float(val) for val in coord_info.requested]
             except ValueError:
                 cmor_points = coord_info.requested
+            if coord.points.ndim != 1:
+                self.report_debug_message(
+                    "Cannot check requested values of {}D coordinate {} since "
+                    "it is not 1D", coord.points.ndim, var_name)
+                return
```

Comment:
I presume this was added because some variable is requested on specific pressure levels in CMIP6, but not delivered on pressure levels in ICON?

Reply:
Yes, ICON only provides 4D pressure levels. We could of course use interpolation in the CMORizer to provide the data on the requested 1D levels, but I don't think that's a good idea since it will add unnecessary errors to the data.